### PR TITLE
Dropdown sasha g

### PR DESCRIPTION
**Pull request by aleksandrgolyk**
Added Dropdown and tested in Header/Layout
image
image
image

### CODE REVIEW

File: src/app/components/Dropdown/Dropdown.template.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  const handleMenu = (event) => {
+    setAnchorEl(event.currentTarget)
+  }
+
+  const handleClose = () => {
+    setAnchorEl(null)
+  }
+  return (
+    <Box>
+      {React.cloneElement(children, { onClick: handleMenu })}
```
Comment (alexpassss) [inline src/app/components/Dropdown/Dropdown.template.js:18]: handleMenu what? Issue with naming. Please discuss naming conventions with others

File: src/app/components/Dropdown/Dropdown.template.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      <Menu
+        id="menu-appbar"
```
Comment (alexpassss) [inline src/app/components/Dropdown/Dropdown.template.js:20]: Id should be static

File: src/app/components/Dropdown/Dropdown.template.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        anchorEl={anchorEl}
+        anchorOrigin={{
+          vertical: 'top',
```
Comment (alexpassss) [inline src/app/components/Dropdown/Dropdown.template.js:23]: These properties should be configurable

File: src/app/components/Dropdown/Dropdown.template.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+          horizontal: 'right'
+        }}
+        keepMounted
+        transformOrigin={{
+          vertical: 'top',
```
Comment (alexpassss) [inline src/app/components/Dropdown/Dropdown.template.js:28]: Same here

File: src/app/components/Header/Header.template.js [hunk 5]
```diff
@@ -1,5 +1,57 @@
...
+import { makeStyles } from '@material-ui/core/styles'
+import AccountCircle from '@material-ui/icons/AccountCircle'
+import NotificationsIcon from '@material-ui/icons/Notifications'
+import Badge from '@material-ui/core/Badge'
+import Dropdown from '../Dropdown/Dropdown.template'
+import { MenuList } from '@material-ui/core'
+import MenuItem from '@material-ui/core/MenuItem'
+import ExitToAppIcon from '@material-ui/icons/ExitToApp'
+
+const useStyles = makeStyles((theme) => ({
```
Comment (alexpassss) [inline src/app/components/Header/Header.template.js:14]: Please, move to the separate file

File: src/app/components/Header/Header.template.js [hunk 6]
```diff
@@ -1,5 +1,57 @@
...
+  root: {
+    alignSelf: 'flex-end'
+  }
+}))
+const DropdownList = (
```
Comment (alexpassss) [inline src/app/components/Header/Header.template.js:19]: Please, move code below to the separate file

File: src/app/components/Header/Header.template.js [hunk 7]
```diff
@@ -1,5 +1,57 @@
...
+    </MenuItem>
+  </>
+)
+
 const Header = (props) => {
-  return <div>Header</div>
+  const classes = useStyles()
+
+  return (
+    <AppBar position="fixed" color="inherit">
```
Comment (alexpassss) [inline src/app/components/Header/Header.template.js:36]: Move styles to the separate file

File: src/app/components/Navbar/Navbar.template.js [hunk 8]
```diff
@@ -1,5 +1,37 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import BottomNavigation from '@material-ui/core/BottomNavigation'
+import BottomNavigationAction from '@material-ui/core/BottomNavigationAction'
+import ShoppingCartIcon from '@material-ui/icons/ShoppingCart'
+import BarChartIcon from '@material-ui/icons/BarChart'
+import SettingsIcon from '@material-ui/icons/Settings'
+import { AppBar } from '@material-ui/core'
+
+const useStyles = makeStyles((theme) => ({
```
Comment (alexpassss) [inline src/app/components/Navbar/Navbar.template.js:10]: Same issue with styles here

**Review by alexpassss — CHANGES_REQUESTED** [review]

File: src/app/components/Dropdown/Dropdown.style.js [hunk 9]
```diff
@@ -1,3 +1,5 @@
-const styles = (theme) => ({})
+import { makeStyles } from '@material-ui/core/styles'
 
-export default styles
+export const useStyles = makeStyles({
+  root: {}
+})
```
Comment (GingerBoy14) [inline src/app/components/Dropdown/Dropdown.style.js]: What the point of empty styles?

File: src/app/components/Navbar/Navbar.template.js [hunk 10]
```diff
@@ -1,14 +1,13 @@
...
 import BottomNavigationAction from '@material-ui/core/BottomNavigationAction'
 import ShoppingCartIcon from '@material-ui/icons/ShoppingCart'
 import BarChartIcon from '@material-ui/icons/BarChart'
 import SettingsIcon from '@material-ui/icons/Settings'
 import { AppBar } from '@material-ui/core'
-import styles from './Navbar.style'
+import { useStyles } from './Navbar.style'
 
-const useStyles = makeStyles(styles)
+// const classes = makeStyles(styles)
```
Comment (GingerBoy14) [inline src/app/components/Navbar/Navbar.template.js:10]: remove comment

**Review by GingerBoy14 — CHANGES_REQUESTED** [review]

**Review by okrdima — APPROVED** [review]

**Review by GingerBoy14 — APPROVED** [review]

**Review by olenashevchuk — APPROVED** [review]

**Pull request merged**